src/components/Accordion.js

```diff
@@ -81,8 +130,12 @@ const menuState = state =>
     state.toLowerCase() === "open" ? "show" : ""
   }`.trim();
 
-const updateButtonStatus = (button, totalCollapsibles, totalCollapsed) => {
-  totalCollapsibles === totalCollapsed
+const updateButtonStatus = (
+  button,
+  totalCollapsiblesPanels,
+  totalCollapsedPanels
+) => {
+  totalCollapsiblesPanels === totalCollapsedPanels
```

Comment:
not blocking now, but this would be better written as

```js
button.textContent = totalCollapsiblesPanels === totalCollapsedPanels ? buttonCloseAll : buttonOpenAll;
```

Which allows you to only set the textCotent in one place, avoiding potential unexpected behavior with duplicated code.